makefiles/helmUtils.mk

```diff
@@ -15,6 +15,7 @@ export DISTROLESSS_FIPS_IMAGE := --set global.baseImage=distroless-fips
 export PHOTON_IMAGE := --set global.baseImage=photon
 export PHOTON_FIPS_IMAGE := --set global.baseImage=photos-fips
 export HELM_CHART_PATH := "./helm-charts/${VERSION}"
+export NAME_TEMPLATE := --name-template vsecm
```

Comment:
Minor, and can be ignored.

Should we consider using `?=` conditional assignment operator. This operator ensures that the NAME_TEMPLATE, for instance, is only set if it hasn't already been defined. This should give better flexibility to the user.

Reference: [MakefileTutorial - Variables, Part 2](https://makefiletutorial.com/#variables-pt-2)

Reply:
We're not expecting any other value @farhan-pasha